style/base.css

```diff
@@ -142,11 +137,11 @@ input::placeholder {
   border-radius: var(--gather-border-radius);
 }
 
-.control-bar.transparent {
+.jlab-gather-control-bar .jlab-gather-transparent {
```

Comment:
This is a double class selector not a descendant selector, so there needs to not be a space.